src/jose/algorithms/types.ts

```diff
@@ -0,0 +1,4 @@
+import type { JwkPrivate, JwkPublic } from '../jwk';
+
+export type Signer = (payload: Uint8Array, privateKeyJwk: JwkPrivate) => Promise<Uint8Array>;
+export type Verifier = (payload: Uint8Array, signature: Uint8Array, publicKeyJwk: JwkPublic) => Promise<boolean>;
```

Comment:
Not seeing the advantage of declaring and exporting the type here since it is only used internally to `jws.ts`.

Supportive of declaring this explicitly however in `jws.ts`.

Reply:
Sure. will go ahead and do that now

Reply:
done